### PR TITLE
Add opacity to background images

### PR DESCRIPTION
**Pull request by thekabistro**
Adding opacity to some background images so they stand out more....

Please check out and leave comments

### CODE REVIEW

**Comment by ryanmao725** [comment]
It seems to really interfere with the right side navbar. The rest of the site seems to look fine.

File: src/Background.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ const imageDefs = [{
     bottom: "-400px",
     right: null,
     height: 1000,
+    opacity: 0.1,
     multiplier: 0.99
 }, {
     source: "./assets/background/square-dotted_white.svg",
```
Comment (shivanmodha) [inline src/Background.jsx]: I think you missed an opacity tag for the largesquaredots (lines 13-26) which is why there is too much noise around the sidenav, as @ryanmao725  pointed out

File: src/Background.jsx [hunk 2]
```diff
@@ -170,7 +183,7 @@ class Background extends Component {
             let images = [];
             for (let i = 0; i < imageDefs.length; i++) {
                 let image = imageDefs[i];
-                images.push(this.renderImage(image.source, image.top, image.left, image.bottom, image.right, image.height, image.transform, image.multiplier ? 1 - image.multiplier : 1));
+                images.push(this.renderImage(image.source, image.top, image.left, image.bottom, image.right, image.height, image.transform, image.multiplier ? 1 - image.multiplier : 1, image.opacity));
```
Comment (shivanmodha) [inline src/Background.jsx]: Tbh maybe we should just expand the entire object

File: src/Background.jsx (hunk 1 above)
Comment (thekabistro) [reply src/Background.jsx]: It's defaulting to 0.25...I'll try reducing it.  It's more the yellow shape that's causing the issue.

Comment (shivanmodha) [reply src/Background.jsx]: O didn't see the default my bad

**Comment by thekabistro** [comment]
See if this is any better....This is a trade off between making the designers happy and having it be visible.

**Pull request merged**